### ISSUE TITLE
Is this supposed to be a Meteor atmosphere package?

### ISSUE THREAD

**Issue by waiholiu** [meteor-typescript/meteor-typescript-libs#9]
Ran meteor add meteortypescript:typescript-libs from the command line to install this.

It didn't put any definition files in the .meteor directory as per your installation instructions

**Comment by fullflavedave** [comment]
I’m sorry to hear that.  So there weren’t any definition files in .meteor, even after you started the server with “meteor” command?  And you are on OSX or Linux and using Meteor 1.0.2 or later?

I’ve attached a screenshot of my terminal commands starting a new sample project in case it’s helpful.

If all else fails, you could just pull down the definitions from GitHub and put them anywhere into your project (https://github.com/meteor-typescript/meteor-typescript-libs/tree/master/definitions <https://github.com/meteor-typescript/meteor-typescript-libs/tree/master/definitions>).  However, I would like to figure out if something is wrong with the package.

Dave







On Jan 16, 2015, at 4:30 AM, waiholiu <notifications@github.com> wrote:

Ran meteor add meteortypescript:typescript-libs from the command line to install this.

It didn't put any definition files in the .meteor directory as per your installation instructions

**Comment by waiholiu** [comment]
Hi.

Yeah there was nothing in the .meteor folder. At least not in the place you
put in step one.

Not sure why. Im on Mac and latest version of meteor.

But yeah I ended up just dragging the definitions down manually.

When I got time I'll start a new project and see if your package works on
that. Might just be my project for some reason


On 17/01/2015 5:48 am, "Dave Allen" <notifications@github.com> wrote:
>
> I’m sorry to hear that. So there weren’t any definition files in .meteor,
even after you started the server with “meteor” command? And you are on OSX
or Linux and using Meteor 1.0.2 or later?
>
> I’ve attached a screenshot of my terminal commands starting a new sample
project in case it’s helpful.
>
> If all else fails, you could just pull down the definitions from GitHub
and put them anywhere into your project (
https://github.com/meteor-typescript/meteor-typescript-libs/tree/master/definitions
<
https://github.com/meteor-typescript/meteor-typescript-libs/tree/master/definitions>).
However, I would like to figure out if something is wrong with the package.
>
> Dave
>
>
>
>
>
>
>
> On Jan 16, 2015, at 4:30 AM, waiholiu <notifications@github.com> wrote:
>
> Ran meteor add meteortypescript:typescript-libs from the command line to
install this.
>
> It didn't put any definition files in the .meteor directory as per your
installation instructions
>
> —
> Reply to this email directly or view it on GitHub <
https://github.com/meteor-typescript/meteor-typescript-libs/issues/9>.
